web_widget_timepicker/__openerp__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2016 Michael Fried @ Vividlab (<http://www.vividlab.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Web Timepicker Widget',
+    'version': '9.0.1.0.0',
+    'author': 'Michael Fried@Vividlab, Odoo Community Association (OCA)',
```

Comment:
this is the first time I see you coming around, so maybe make this only your company? What you fill in here will show up in the app store, so for visibility, you'll probably want to choose one unique name for all the modules you contribute and not employee@company